plugins/jobtype/src/azkaban/jobtype/HadoopSparkJob.java

```diff
@@ -175,6 +188,35 @@ public HadoopSparkJob(String jobid, Props sysProps, Props jobProps, Logger log)
     }
   }
 
+  /**
+   * Add additional namenodes specified in the Spark Configuration
+   * ({@link #SPARK_CONF_ADDITIONAL_NAMENODES}) to the Props provided.
+   * @param props Props to add additional namenodes to.
+   * @see HadoopJobUtils#addAdditionalNamenodesToProps(Props, String)
+   */
+  void addAdditionalNamenodesFromConf(Props props) {
```

Comment:
Can we use the method getSparkProperties inside HadoopSecureSparkWrapper to get this config param we need?
We can change the scope of that method to make it accessible here.
No need to duplicate the logic for fetching configs inside spark-defaults.conf.

Also, once the new Azkaban artifact becomes available in artifactory, make sure to also send out an RB to update the internal Spark job type repo.